ede.js

```diff
@@ -27,7 +28,11 @@
         let apiPrefix = 'https://ddplay-api.930524.xyz/cors/';
         let logQueue = [];
         let logLines = 0;
-        const baseUrl = window.location.origin + window.location.pathname.replace('/web/index.html', '');
+        const jellyfinCredentials = JSON.parse(localStorage.getItem('jellyfin_credentials'));
+        let baseUrl = jellyfinCredentials.Servers[0].ManualAddress;
+        if (!baseUrl) {
+            baseUrl = window.location.origin + window.location.pathname.replace('/web/index.html', '');
+        }
```

Comment:
You can try `ApiClient.serverAddress()`, `ApiClient.accessToken()`, and `ApiClient.getCurrentUserId()`. `ApiClient.getSessions()` also helps.

Reply:
Thank you for your suggestion. `ApiClient` is better. I will update the script with it later.